test/runtime/chaos.go

```diff
@@ -109,4 +110,14 @@ var _ = Describe("RuntimeChaos", func() {
 		Expect(links).Should(Equal(originalLinks),
 			"Some network interfaces were accidentally removed!")
 	}, 300)
+
+	It("Checking for file-descriptor leak", func() {
+		threshold := 5000
+		fds, err := vm.Exec("sudo lsof -p `pidof cilium-node-monitor` -p `pidof cilium-agent` -p `pidof cilium-docker` 2>/dev/null | wc -l").IntOutput()
+		Expect(err).Should(BeNil())
+
+		aboveThreshold := fds > threshold
+		Expect(aboveThreshold).Should(Equal(false),
```

Comment:
I don't really like this approach, what about `Expect(fds).To(BeNumerically("<", threshold))` and you'll get the values in the assert message. 

Regards

Reply:
I was looking for something like that, but couldn't find it. Thanks for that, I prefer what you suggested as well :-)

Reply:
Fixed in latest upload.